govuk_connect.gemspec

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'govuk_connect/version'
+
+Gem::Specification.new do |s|
+  s.name          = 'govuk_connect'
+  s.version       = GovukConnect::VERSION
+
+  s.summary       = 'govuk-connect CLI tool'
+  s.description   = 'CLI tool to connect to GOV.UK infrasturcture'
```

Comment:
While I know CLI just means command line interface, and this does apply to `govuk-connect`, there's been plenty of controversy around referring to command line interfaces thusly (I did briefly create a `gouvk-cli` repository for `govuk-connect`, but it proved too controversial).

So maybe say "Command line tool ...", or just "Tool ..." to avoid any of this?

Reply:
Ahah okay - I've changed to "command line tool" in the description and summary